scss/_navbar.scss

```diff
@@ -203,6 +184,18 @@
         .navbar-toggler {
           display: none;
         }
+
+        .dropup {
+          .dropdown-menu {
+            position: absolute;
+            top: auto;
+            bottom: 100%;
+          }
+
+          .dropdown-toggle {
```

Comment:
This is not necessary. `.dropdown-toggle` is already in `_dropdown.scss`. Nothing changes here.